bb/viewer_resources_hashing.clj

```diff
@@ -0,0 +1,71 @@
+(ns viewer-resources-hashing
+  (:require [babashka.classpath :as cp]
+            [babashka.fs :as fs]
+            [babashka.tasks :as tasks :refer [shell]]
+            [clojure.string :as str]
+            [nextjournal.dejavu :as djv]))
+
+(def output-dirs ["resources/public/ui"
+                  "resources/public/build"])
+
+;; Example link in bucket:
+;; "https://storage.googleapis.com/nextjournal-cas-eu/data/8VwKauX6JACEP3K6ahNmP5p1w7rWdhKzeGXCDrHMnJiVrUxHVxcm3Xj84K2r3fcAKWxMQKzqoFe92osgFEHCuKCtZC"
+
+(def gs-bucket "gs://nextjournal-cas-eu/data")
+(def base-url "https://storage.googleapis.com/nextjournal-cas-eu/data")
```

Comment:
I think we should drop the `/data` prefix from the base-url and gs-bucket and create the lookup files on the top-level.